services/explorer/src/pages/AddressInfo/AddressInfo.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Box } from '@mui/material';
+import { AddressInfo } from '@taraxa_project/taraxa-ui';
+import { PageTitle } from '../../components';
+import { useAddressInfoEffects } from './AddressInfo.effects';
+
+const AddressInfoPage = () => {
```

Comment:
Can you reuse the AddressInfo component and BlockTable for the other pages as well (Transactions and Blocks)?. It probabbly needs some customization but I think it will really help.

Reply:
Yep, if we extract it to the Taraxa UI we need to rename it so that the naming corresponds with its purpose: reuse to display `detailed data about transactions, blocks and addresses` : dunno what to call it, but it will definitely need some prop changes. As I foresee, besides these 3 categories there will be another data container for smart contracts, just like on Etherscan, but maybe you could refactor it to have 3 variants: transaction, block or address, and then handle the display discrepancies internally.